src/commands/general/salty.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const salty: CommandDefinition = {
+    name: ['salty', 'sal'],
+    description: 'Provides link to salty discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/S4PJDwk');
+        await msg.delete();
+    },
+};
```

Comment:
```suggestion
};

```

Add new line at the end